docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -49,6 +49,30 @@ In v2.7.0, the built-in extensions will not be displayed under the **Available**
 
     ![Reload button](/img/reload-button.png)
 
+### Importing and Installing Extensions in an Air-Gapped Environment
+
+1. Find the address of the container image you want to import as an extension.
+
+1. [Create](../how-to-guides/new-user-guides/kubernetes-resources-setup/secrets.md) a registry secret within the `cattle-UI-plugin-system` namespace. Enter the domain of the image address in the **Registry Domain Name** field.
```

Comment:
There is not always a secret needed. For example, for publicly available image such as https://hub.docker.com/r/rancher/ui-plugin-catalog/tags, there is no need to secret to be created. Can we mention this somehow ?